### PR TITLE
Feature/801 fe editorial events landing page

### PR DESCRIPTION
**Pull request by nicklee**
New editorial and events landing page, includes a small refactor to the news listing to allow for four items in each feed (previously limited to three items).

Ticket:
https://projects.torchbox.com/projects/rca-website-rebuild/tickets/801

Dev URL (currently syncing):
https://rca-development.herokuapp.com/pattern-library/render-pattern/patterns/pages/editorial_event_landing/editorial_event_landing.html

Screenshots:

Screenshot 2021-07-16 at 11 11 26

Screenshot 2021-07-16 at 11 11 20

### CODE REVIEW

File: rca/project_styleguide/templates/patterns/pages/editorial_event_landing/editorial_event_landing.html [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+{% endblock %}
+
+{% block content %}
+
+    <div class="page page--no-hero">
+
+        <header class="page__header bg bg--dark">
+            <div class="title-area title-area--landing-page grid">
+                <div class="title-area__content">
+                    {% if page.title %}
```
Comment (chris-lawton) [inline rca/project_styleguide/templates/patterns/pages/editorial_event_landing/editorial_event_landing.html:16]: should this wrap the entirety of the header or is it still needed even if there's no page title for visual reasons?

File: rca/project_styleguide/templates/patterns/pages/editorial_event_landing/editorial_event_landing.html [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+                    </div>
+                    <div class="section__content grid">
+                        {% include "patterns/organisms/news/news_api_content.html" with news=page.news news_view_all=page.news_view_all modifier="news--single-feature" %}
+                    </div>
+                </div>
+            </section>
+
+            {# Events #}
+            <section class="section section--start anchor-heading bg bg--dark" id="events">
+                <div class="section__row section__row--last schools__about--news">
```
Comment (chris-lawton) [inline rca/project_styleguide/templates/patterns/pages/editorial_event_landing/editorial_event_landing.html:42]: should `schools__about--news` be `schools__about--events`?

Comment (chris-lawton) [reply rca/project_styleguide/templates/patterns/pages/editorial_event_landing/editorial_event_landing.html:42]: ah just seen a few other news related bits in this section (`modifier="news--single-feature"` on line 47) so perhaps they're all being shared

File: rca/static_src/sass/components/_news.scss [hunk 3]
```diff
@@ -56,8 +56,21 @@
...
+                        border-bottom: 1px solid $color--grid-line-light;
+                    }
+                }
+            }
+        }
+
+        &--single-feature {
+            .card {
+                &:nth-child(2) {
+                    @include media-query(large) {
```
Comment (chris-lawton) [inline rca/static_src/sass/components/_news.scss:73]: that's some deep nesting : )

**Review by chris-lawton — APPROVED** [review]

File: rca/project_styleguide/templates/patterns/pages/editorial_event_landing/editorial_event_landing.html (hunk 1 above)
Comment (nicklee) [reply rca/project_styleguide/templates/patterns/pages/editorial_event_landing/editorial_event_landing.html:16]: yep good plan!

File: rca/project_styleguide/templates/patterns/pages/editorial_event_landing/editorial_event_landing.html (hunk 2 above)
Comment (nicklee) [reply rca/project_styleguide/templates/patterns/pages/editorial_event_landing/editorial_event_landing.html:42]: we can ditch the school classes, copied from the schools template and forgot to remove, good spot :)

**Pull request merged**